buku

```diff
@@ -2784,14 +2784,14 @@ def convert_bookmark_set(
     out = ''
     if export_type == 'markdown':
         for row in resultset:
-            if row[2] == '':
+            if not row[2]:
```

Comment:
I don't even remember these cases. If they are working, please don't touch.

Reply:
I think this was because we never have None/NULL in any field. It's always an empty string.

Reply:
```
1438     def test_exportdb_to_db():
1439         with NamedTemporaryFile(delete=False) as f1, NamedTemporaryFile(delete=False, suffix='.db') as f2:
1440             db = BukuDb(dbfile=f1.name)
1441             db.add_rec('http://example.com')
1442             db.add_rec('http://nonexit.org')
1443             db.add_rec('http://google.com')
1444             __import__('pdb').set_trace()
(Pdb++) db.get_rec_all()
[(1, 'http://example.com', 'Example Domain', ',', '', 0), (2, 'http://nonexit.org', None, ',', '', 0), (3, 'http://google.com', 'Google', ',', '', 0)]
```
it will output for nonexistent site (or maybe for 404 or other site) but at least it is possible that `None` is returned